# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Fri Oct 23 14:02:54 2020 -0400
@@ -21,15 +21,13 @@
 
 #include "pk11pub.h"
 
-#include "webrtc/modules/audio_coding/codecs/builtin_audio_decoder_factory.h"
-#include "webrtc/modules/audio_coding/codecs/builtin_audio_encoder_factory.h"
-
-#include "webrtc/modules/audio_processing/include/audio_processing.h"
-#include "webrtc/modules/rtp_rtcp/include/rtp_rtcp.h"
-#include "webrtc/modules/rtp_rtcp/source/rtp_packet_received.h"
-#include "webrtc/voice_engine/include/voe_errors.h"
-#include "webrtc/voice_engine/voice_engine_impl.h"
-#include "webrtc/system_wrappers/include/clock.h"
+#include "api/audio_codecs/builtin_audio_decoder_factory.h"
+#include "modules/audio_processing/include/audio_processing.h"
+#include "modules/rtp_rtcp/include/rtp_rtcp.h"
+#include "modules/rtp_rtcp/source/rtp_packet_received.h"
+#include "modules/voice_engine/include/voe_errors.h"
+#include "modules/voice_engine/voice_engine_impl.h"
+#include "system_wrappers/include/clock.h"
 
 #ifdef MOZ_WIDGET_ANDROID
 #  include "AndroidBridge.h"